pages/jobs/index.tsx

```diff
@@ -270,11 +270,11 @@ export async function getServerSideProps(context) {
 		(res) => res.json()
 	);
 
-	const project = await fetch(
-		`${getAPIOrigin(context.req)}/api/project`
+	const projects = await fetch(
+		`${getAPIOrigin(context.req)}/api/projects`
```

Comment:
When you nest this file under a dynamic projects directory, you will be able to get the id from the url via
```js
const {projectId} = context.req.query
```